pandora-client-web/src/ui/components/chat/commands.ts

```diff
@@ -321,6 +321,94 @@ export const COMMANDS: readonly IClientCommand<ICommandExecutionContextClient>[]
 				return true;
 			}),
 	},
+	{
+		key: ['cards'],
+		description: 'Play a game of cards',
+		longDescription: `You can 'create' or 'stop' a game of cards, 'join' an existing one, 'deal' cards open or face down to player or the room,
+		'check' your hand or 'reveal' the cards that were dealt to all players, ending the game.`,
+		usage: 'create | stop | join | deal <target> /hidden | check | reveal',
+		handler: CreateClientCommand()
+			.fork('action', (ctx) => ({
+				create: {
+					description: 'Create a new game with a deck of 52 cards for the current space.',
+					handler: ctx
+						.handler(({ shardConnector }) => {
+							shardConnector.sendMessage('gamblingAction', {
+								type: 'cards',
+								msgOption: 'create',
+							});
+							return true;
+						}),
+				},
+				stop: {
+					description: 'Stop an ongoing game. Only possible for the creator of the game.',
+					handler: ctx
+						.handler(({ shardConnector }) => {
+							shardConnector.sendMessage('gamblingAction', {
+								type: 'cards',
+								msgOption: 'stop',
+							});
+							return true;
+						}),
+				},
+				join: {
+					description: 'Join an ongoing game.',
+					handler: ctx
+						.handler(({ shardConnector }) => {
+							shardConnector.sendMessage('gamblingAction', {
+								type: 'cards',
+								msgOption: 'join',
+							});
+							return true;
+						}),
+				},
+				deal: {
+					description: `Deal one card from the space's deck to a player or the room. Only possible for the game creator.`,
+					handler: ctx
+						.argument('target', CommandStepOptional(CommandSelectorCharacter({ allowSelf: 'any' })))
+						.argument('option', CommandSelectorEnum(['', '/hidden']))
+						.handler(({ shardConnector }, { target, option }) => {
+							if (target) { //Deal card to a player
+								shardConnector.sendMessage('gamblingAction', {
+									type: 'cards',
+									targetId: target.data.id,
+									msgOption: 'deal',
+									dealHidden: option === '/hidden',
+								});
+							} else { //Deal card to room
+								shardConnector.sendMessage('gamblingAction', {
+									type: 'cards',
+									msgOption: 'deal',
+									dealHidden: false, // Cards to the room are always dealt openly
+								});
+							}
+							return true;
+						}),
+				},
```

Comment:
After playing with this command for a while, it feels rather confusing to use. My three main points are:
- It is not apparent that not using a target is a possibility (this comes from how `CommandStepOptional` works)
- The `/hidden` is actually most common, yet it is easy to accidentally not specify it
- It shows characters that are not part of the game (this unfortunately cannot be helped without syncing some data in advance)

I would suggest splitting this subcommand into three subcommands to take care of the first two steps: `dealTable`, `dealOpenly` and `dealHidden` (or simply `deal` as it is most common).